packages/tasks/src/model-libraries.ts

```diff
@@ -275,6 +275,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "mindspore",
 		repoUrl: "https://github.com/mindspore-ai/mindspore",
 	},
+	"mars5-tts": {
```

Comment:
Given it's a simple snippet (`mars5 = Mars5TTS.from_pretrained("CAMB-AI/MARS5-TTS")`) maybe add it as well

Reply:
Added in the recent commit